pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -111,9 +111,7 @@ let
     inherit lib stdenv version;
     rustAvailable =
       lib.any (lib.meta.platformMatch stdenv.hostPlatform) rustc.targetPlatforms
-      && lib.all (p: !lib.meta.platformMatch stdenv.hostPlatform p) rustc.badTargetPlatforms
-      # Known to be broken: https://lore.kernel.org/lkml/31885EDD-EF6D-4EF1-94CA-276BA7A340B7@kernel.org/T/
-      && !(stdenv.hostPlatform.isRiscV && stdenv.cc.isGNU);
+      && lib.all (p: !lib.meta.platformMatch stdenv.hostPlatform p) rustc.badTargetPlatforms;
```

Comment:
This is just `availableOn` now, right?

Reply:
I suppose so, it certainly looks like it. Will fix in the next revision.